src/static/styles/partials/_playlists.css

```
@@ -232,3 +243,408 @@
+  transition: color 0.15s ease, border-color 0.15s ease;
+}
+
+.pl-add-row:hover,
+a.pl-add-row:hover {
+  color: var(--accent);
+  border-color: var(--accent);
+}
+
+/* Visual parity pass: restore the handoff's rotation-card header/body split
+   and visible desktop controls instead of icon-only admin actions. */
+.playlist-frame {
```

Stylelint (17.7.0)

[error] Duplicate selector ".playlist-frame", first used at line 192 (no-duplicate-selectors)

(no-duplicate-selectors)